Wrapping/Generators/Python/itk/support/extras.py

```diff
@@ -385,6 +385,80 @@ def GetImageViewFromArray(arr, is_vector: bool = False, ttype=None):
 image_view_from_array = GetImageViewFromArray
 
 
+def array_from_vector_container(container, ttype=None):
+    """Get an Array with the content of the vector container"""
+    import itk
+
+    # Find container type
+    if ttype is not None:
+        if isinstance(ttype, (tuple, list)):
+            if len(ttype) != 1:
+                raise RuntimeError("Expected 1 component in ttype tuple.")
+            DataType = ttype[0]
+        else:
+            DataType = ttype
+    else:
+        DataType = itk.template(container)[1][0]
+    keys = [k for k in itk.PyVectorContainer.keys() if k[0] == DataType]
+    if len(keys) == 0:
+        raise RuntimeError("No suitable template parameter can be found.")
+    # Create numpy array of the type of the input container
+    return itk.PyVectorContainer[keys[0]].array_from_vector_container(container)
+
+
+# FIXME
+#  >>> itk.array_from_vector_container(x)
+# Traceback (most recent call last):
+#  File "<stdin>", line 1, in <module>
+#  File "C:\repos\ITK-build\Wrapping\Generators\Python\itk\support\itkExtras.py", line 328, in array_from_vector_container
+#    return itk.PyVectorContainer[keys[0]].array_view_from_vector_container(container)
+#  File "C:\repos\ITK-build\Wrapping\Generators\Python\itk\itkPyVectorContainerPython.py", line 622, in array_view_from_vector_container
+#    memview       = itkPyVectorContainerUC._array_view_from_vector_container(vector_container)
+# TypeError: in method 'itkPyVectorContainerUC__array_view_from_vector_container', argument 1 of type 'itkVectorContainerULLUC *
+
+
```

Comment:
Is this error still arising even with your fix?

Reply:
Hi Lee, this error is the same as the one above where `PyVectorContainer` is only wrapped to accept `VectorContainer`s with index type `unsigned long long` (aka `Py_ssize_t`). Per discussion above the best plan is probably to expose the `numpy` Python interface in this PR with existing wrapping and then add additional wrapping to `PyVectorContainer` in a subsequent PR.

I'll go ahead and remove the inline comment as it is now outdated. The error still occurs but is not a blocking case for this PR.

Reply:
Got it.  It looks like VectorContainer needs wrapping changes too; `itkVectorContainer.wrap` wrapping supports `${ITKM_IT}` as its first template argument but not "ULL" too.

Reply:
Apologies, `PyVectorContainer` takes in `IdentiferType` which is equivalent to `uint64_t` and `Py_ssize_t` but may not be equivalent to `unsigned long long` depending on the system. It is correct to  wrap `VectorContainer` with `${ITKM_IT}`. I've updated my above comment accordingly.

What is the best way to discern system wrapping types? For instance, on my system `${ITKM_IT}` seems to wrap for `itk.ULL` but on other systems I understand it may wrap for `itk.UL`.

Reply:
I don't know how to get "UL" or "ULL" from `Py_ssize_t` or `IdentifierType`.  Maybe you can get at the parameter types of one of the functions:
```python
itk.PyVectorContainer[itk.F].array_view_from_vector_container
itk.PyVectorContainer[itk.F]._array_view_from_vector_container
```
etc., and use that.

Reply:
Just put in [Draft PR 2443](https://github.com/InsightSoftwareConsortium/ITK/pull/2443) which tackles typing issues discussed here. Going to go ahead and resolve this thread but please feel free to add comments there.